ios/RCTTWVideoModule.m

```diff
@@ -471,10 +471,10 @@ -(NSMutableDictionary*)convertLocalVideoTrackStats:(TVILocalVideoTrackStats *)st
       builder.preferredVideoCodecs = @[ [TVIH264Codec new] ];
     }
 
-    if(encodingParameters[@"audioBitrate"] || encodingParameters[@"videoBitrate"]){
-      NSInteger audioBitrate = [encodingParameters[@"audioBitrate"] integerValue];
```

Comment:
so we could already pass audio and video bitrate?

Reply:
if we can pass it from RN, then it is better, sure

Reply:
done